rust/experimental/server/src/app.rs

```diff
@@ -183,6 +183,13 @@ impl App {
     }
 
     pub async fn select(&self, ctx: ReadingViewContext) -> Result<ResponseData, WorkerError> {
+        match ctx.reading_options {
+            ReadingOptions::MEMORY_LAST_BLOCK_ID_AND_MAX_SIZE(_, length)
+            | ReadingOptions::FILE_OFFSET_AND_LEN(_, length) => {
+                TOTAL_READ_DATA.inc_by(length as u64);
+            }
+        }
+
         self.store.get(ctx).await
```

Comment:
Why not extract the responseData to get the read size? The current change is not accurate, especially the reading is slow, and then the timeserial metrics will be incorrect.

Reply:
make sense